.github/config/android-arm.json

```diff
@@ -13,7 +13,8 @@
         "gpu_unittests",
         "gin_unittests",
         "blink_unittests",
-        "content_shell",
+        "media_unittests",
+        "content_shell_apk",
```

Comment:
Why are we changing `content_shell` to `content_shell_apk`? I would prefer a target name that is shared between platforms, which `content_shell` is.

Reply:
For android targets, it should be `xxx_apk`, not just `xxx`.

Reply:
Fixed.